charts/garnet/templates/cmd-pod.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cmd
+  namespace: garnet
+  labels:
+    app: cmd
+spec:
+  containers:
+  - name: cmd
+    image: redis:latest
```

Comment:
consider allowing to override this - similar to the `init-job` configuration (`"{{ .Values.cluster.initJob.image.registry }}/{{ .Values.cluster.initJob.image.repository }}:{{ .Values.cluster.initJob.image.tag | default "latest" }}"`)

Reply:
How was this addressed, I don't see a new commit. Thanks.